content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
@tinaafitz oVirt always creates a VM at the beginning of V2V process. But V2V that converts the VM and refresh that puts VM into CFME's inventory are two independent processes. And my and Martin Betak's testing shows that sometimes the VM appears before and sometimes after the V2V process completes.